app/controllers/test_messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class TestMessagesController < ApplicationController
+  def new
+    @message = ScheduledMessage.find(params[:scheduled_message_id])
+  end
+
+  def create
+    message = ScheduledMessage.find(params[:scheduled_message_id])
+    employee = Employee.find_by(slack_username: slack_username)
+
+    if employee
+      MessageSender.new(employee, message).delay.run
```

Comment:
We want these to run async (with Delayed Job) so the request doesn't time out waiting for the messages to send (this might happen if the Slack API is down or being slow)

I also saw this wasn't working yesterday...and the error message is "not_authed" which I am pretty sure means DJ isn't seeing the environment variables that allow it to auth with Slack (the API key). Will look into this more now.